laravel/app/Models/EvaluateData.php

```diff
@@ -18,5 +18,18 @@ class EvaluateData extends Model
      *
      * @var bool
      */
-    public $timestamps = false;
+    public $timestamps = true;
+    
+    protected $fillable = [
+        "date", 
+        "location", 
+        "diagnosis", 
+        "procedure", 
+        "ASA", 
+        "resident_id", 
+        "resident", 
+        "attending_id", 
+        "attending", 
+        "time_with_attending"
+    ];
```

Comment:
When this many things are `fillable`, it's a good sign that `guarded` might be an easier choice.